### ISSUE TITLE
the tf2crf supports multi-label classification?

### ISSUE THREAD

**Issue by ktakanov** [xuxingya/tf2crf#33]
the tf2crf supports multi-label classification?

**Comment by xuxingya** [comment]
I think you don't need crf to do classification task.

**Comment by ktakanov** [comment]
Can you deploy an example of CRF implementation use sparse_target=False?

**Comment by xuxingya** [comment]
You need to one-hot encoding your y_true. For example:
If you have n classes, and y_true is the true labels with shape
(batch_size, sequence_len):
```
from tf.keras.preprocessing.text import one_hot
y_yrue = one_hot(y_true, n)
```
